server/api/index.js

```diff
@@ -1,31 +1,35 @@
-const Shopify = require("shopify-api-node");
-const express = require("express");
-const axios = require("axios");
-
-const { SHOPIFY_SHOP_NAME, SHOPIFY_API_KEY, SHOPIFY_PASSWORD } = process.env;
+const Shopify = require('shopify-api-node');
+const express = require('express');
 const router = express.Router();
+const { SHOPIFY_SHOP_NAME, SHOPIFY_API_KEY, SHOPIFY_PASSWORD } = process.env;
 
 const shopify = new Shopify({
-  shopName: SHOPIFY_SHOP_NAME,
-  apiKey: SHOPIFY_API_KEY,
-  password: SHOPIFY_PASSWORD
+	shopName: SHOPIFY_SHOP_NAME,
+	apiKey: SHOPIFY_API_KEY,
+	password: SHOPIFY_PASSWORD,
 });
 
-shopify.on("callLimits", limits => console.log(limits, "test2"));
+const callShopify = () => {
+	const currentDate = new Date(); // Return orders only form current day
+	shopify.order
+		.list({ financial_status: 'paid', created_at_min: currentDate })
+		.then(orders => {
+			const totalPrice = orders.reduce((acc, value, index) => {
+				return acc + Number(value.total_price);
+			}, 0);
+			console.log(totalPrice);
+		})
+		.catch(err => console.error(err));
+};
+
+const timer = 30000;
+
+setInterval(callShopify, timer);
 
-router.post("/transaction", async (req, res) => {
-  console.log(req.body);
-  const url = "https://weglimpse.co/admin/api/2019-10/reports.json";
-  try {
-    const { data } = await axios.get(url);
+router.post('/transaction', async (req, res) => {
+	console.log(req.body.totalPrice);
```

Comment:
Did you test that totalPrice exists on `req.body`?

Reply:
Yes after reviewing!